rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml

```diff
@@ -0,0 +1,115 @@
+            - '\powershell_ise.exe'
+    selection_special_child_powershell_cli:
+        - CommandLine|contains:
+            - ' echo '
+            - '-dumpmode'
+            - '-ssh'
+            - '.dmp'
+            - 'add-MpPreference'
+            - 'adscredentials'
+            - 'bitsadmin'
+            - 'certutil'
+            - 'csvhost.exe'
+            - 'DownloadFile'
+            - 'DownloadString'
+            - 'dsquery'
+            - 'ekern.exe'
+            - 'FromBase64String'
+            - 'iex '
+            - 'iex('
+            - 'Invoke-Expression'
+            - 'Invoke-WebRequest'
+            - 'localgroup administrators'
+            - 'net group'
+            - 'net user'
+            - 'o365accountconfiguration'
+            - 'query session'
+            - 'samaccountname='
+            - 'set-MpPreference'
+            - 'svhost.exe'
+            - 'System.IO.Compression'
+            - 'System.IO.MemoryStream'
+            - 'usoprivate'
+            - 'usoshared'
+            - 'whoami'
+        - CommandLine|re: '[-/–][Ee^]{1,2}[ncodema^]*\s[A-Za-z0-9+/=]{15,}'
+    selection_special_child_lsass_1:
+        CommandLine|contains: 'lsass'
+    selection_special_child_lsass_2:
+        CommandLine|contains:
+            - 'procdump'
+            - 'tasklist'
+            - 'findstr'
+    selection_child_wget:
+        Image|endswith: '\wget.exe'
+        CommandLine|contains: 'http'
+    selection_child_curl:
+        Image|endswith: '\curl.exe'
+        CommandLine|contains: 'http'
+    selection_child_script:
+        CommandLine|contains:
+            - 'E:jscript'
+            - 'e:vbscript'
+    selection_child_localgroup:
+        CommandLine|contains|all:
+            - 'localgroup Administrators'
+            - '/add'
+    selection_child_net:
+        CommandLine|contains:
+            - 'net' # Covers net1
+        CommandLine|contains|all:
+            - 'user'
+            - '/add'
+    selection_child_reg:
+        - CommandLine|contains|all:
+            - 'reg add'
+            - 'DisableAntiSpyware'
+            - '\Microsoft\Windows Defender'
+        - CommandLine|contains|all:
+            - 'reg add'
+            - 'DisableRestrictedAdmin'
+            - 'CurrentControlSet\Control\Lsa'
+    selection_child_wmic_1:
+        CommandLine|contains|all:
+            - 'wmic'
+            - 'process call create'
+    selection_child_wmic_2:
+        CommandLine|contains|all:
+            - 'wmic'
+            - 'delete'
+            - 'shadowcopy'
+    selection_child_vssadmin:
+        CommandLine|contains|all:
+            - 'vssadmin'
+            - 'delete'
+            - 'shadows'
+    selection_child_wbadmin:
+        CommandLine|contains|all:
+            - 'wbadmin'
+            - 'delete'
+            - 'catalog'
+    condition: all of selection_parent_path and (all of selection_special_child_powershell_* or all of selection_special_child_lsass_* or 1 of selection_child_*)
```

Comment:
the test_rules.py doesn't work for this folder. Also there is no `selection_parent_path` in this rule. :)

Reply:
We have to rework the repo a bit. I just started thinking about the solution. Seems to me it the straight forward solution is to duplicate the workflow action and run a separate job for the emerging threats folder

Reply:
I actually reworked the tests here https://github.com/SigmaHQ/sigma/pull/4161
I won't merge this if flo doesn't merge it now. I'm still migrating the older rules.

Reply:
Don't mind the shit show that happened when I did a git pull from master (big mistake)

Reply:
did you do a --rebase pull? imo that's the preferred way to do so.

You cannot merge yet, you have the error in the condition of the first rule. one moment

Reply:
Alright a rebase has done the job. Cheers.

Reply:
Other PR merged. We'll merge this.